machines/README.md

```diff
@@ -0,0 +1,26 @@
+# Vagrantfiles for WSO2 Identity Server
+
+This section defines the procedure to execute run Vagrant resources for a setup of WSO2 Identity Server single
+node with Analytics support.<br>
+
+Please note that in order to run these Vagrant resources use, you need to install
+[Oracle VM VirtualBox](http://www.oracle.com/technetwork/server-storage/virtualbox/downloads/index.html),
+as Vagrant uses Oracle VM VirtualBox as the default provider.
+
+## How to run the Vagrantfiles
```

Comment:
Same as above, might need to be singular.

Reply:
+1